.github/workflows/ubuntu-system-deps-build.yml

```diff
@@ -6,6 +6,7 @@ on:
       - main
       - develop
       - release/*
+      - *
```

Comment:
Yes. I should have put a comment. It's to trigger the action for this branch and check that the action does pass.
Since it's green I will now remove this commit